php.ini

```diff
@@ -0,0 +1,5 @@
+log_errors = On
+expose_php = Off
+display_errors = Off
+error_reporting = E_ALL
+session.save_path = "${PWD}/server/sessions"
```

Comment:
if you have a lot of users this could fill up your ec2 harddrive but it would take a while since you arent storing much